### PR TITLE
Modified TAP beta2 docs

### PR DESCRIPTION
**Pull request by dhadukk**

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -84,58 +89,54 @@ To add the Tanzu Application Platform package repository:
       cnrs.tanzu.vmware.com              Cloud Native Runtimes                     Cloud Native Runtimes is a serverless runtime based on Knative
     ```
 
-7. List version information for the `cnrs.tanzu.vmware.com` package by running:
+
+<<<<<<< HEAD
+
+=======
+>>>>>>> Tanzu CLI updates
```
Comment (divmadan) [inline install.md]: Resolve conflict

**Comment by jncd** [comment]
Hi @dhadukk 
So strange, it looks like we've got a merge conflict again. Please take a look. I've been merging all the PRs quickly now so I'm not quite sure how this happens.

File: install.md [hunk 2]
```diff
@@ -84,58 +89,54 @@ To add the Tanzu Application Platform package repository:
...
 
-
-## <a id='general-procedure-to-install-a-package'></a> General Procedure to Install a Package
-
-To install any package from the Tanzu Application Platform package repository:
-
+2. Run:
+=======
 1. Run:
+>>>>>>> Tanzu CLI updates
```
Comment (divmadan) [inline install.md]: Resolve conflict

**Review by divmadan — CHANGES_REQUESTED** [review]

File: install-general.md [hunk 3]
```diff
@@ -362,25 +362,32 @@ To install the Tanzu CLI on a Windows operating system:
...
-1. (Optional) Remove existing plugins from any previous CLI installations.
+If you have any previous Tanzu CLI installed please run below steps before moving on to [Install the Tanzu CLI Plugins](#Install the Tanzu CLI Plugins).
+1.  Set TANZU_CLI_NO_INIT=true
+2.  export TANZU_CLI_NO_INIT=true
+3.  Run this command to make sure the default plugin repo points to the right path 
+    ```
+    tanzu plugin repo update -b tanzu-cli-framework core
+    ```
 
+4.  Remove existing plugins from any previous CLI installations.
```
Comment (divmadan) [inline install-general.md:375]:
```suggestion
1.  Set the environment variable `TANZU_CLI_NO_INIT`.
     ```
     export TANZU_CLI_NO_INIT=true
     ```

2.  Run this command to make sure the default plugin repo points to the right path:
    ```
    tanzu plugin repo update -b tanzu-cli-framework core
    ```

3.  Remove existing plugins from any previous CLI installations.

Comment (dhadukk) [reply install-general.md]: @jncd  Please merge it now. I have resolved all the conflicts. I think conflicts were due to some PRs merged before mine.

**Comment by dhadukk** [comment]
@jncd Please merge it now. I have resolved all the conflicts. I think conflicts were due to some PRs merged before mine.

**Comment by jncd** [comment]
Thanks! @dhadukk
i'm going to merge this right away. Before anyone else makes changes. 😁 
@divmadan I see that you're a reviewer. Just make additional PRs if you want to adjust this new text further.

**Comment by dhadukk** [comment]
Thanks a lot @jncd

**Comment by a-peek4** [comment]
@dhadukk Can you explain what the "Tanzu core executable" is?

**Pull request merged**